### PR TITLE
GameDB: some title fixes letter A + SCAJ-30004 + SLPM-62346

### PR DESCRIPTION
**Pull request by rufotheone-fr**
### Description of Changes
Some title fixes.

### Rationale behind Changes
GameDB accuracy.

### Suggested Testing Steps
...

### CODE REVIEW

**Comment by rufotheone-fr** [comment]
BTW, is there a process to add unlicensed titles?
For example: [Game Studio Version 1.03a](http://redump.org/disc/80140/)

**Comment by rufotheone-fr** [comment]
Is my PR missing something that prevent it to be merged?

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -27570,7 +27570,7 @@ SLPM-65576:
   name: "Tantei Jinguuji Saburou - Kind of Blue"
   region: "NTSC-J"
 SLPM-65579:
-  name: "After School Love Beat"
+  name: "Simple 2000 Series Ultimate Vol. 27 - Houkago no Love Beat"
```
Comment (Tokman5) [inline bin/resources/GameIndex.yaml:27573]: SLPM-65579 is the original version of ``Houkago no Love Beat``, not the one of Simple series.

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -28072,7 +28072,7 @@ SLPM-65731:
   name: "Tom Clancy's Rainbow Six 3"
   region: "NTSC-J"
 SLPM-65732:
-  name: "Akaiito"
+  name: "SuperLite 2000 Vol. 34 - Akai Ito"
```
Comment (Tokman5) [inline bin/resources/GameIndex.yaml:28075]: SLPM-65732 is the original version of ``Akai Ito``

File: bin/resources/GameIndex.yaml (hunk 1 above)
Comment (rufotheone-fr) [reply bin/resources/GameIndex.yaml:27573]: Both Houkago no Love Beat and Akai Ito Simple/SuperLite 2000 versions got a new serial number on their boxes but had the same disc as original versions so both are fine.

File: bin/resources/GameIndex.yaml (hunk 2 above)
Comment (rufotheone-fr) [reply bin/resources/GameIndex.yaml:28075]: Reverted to base title

**Comment by JordanTheToaster** [comment]
Use the proper naming for commits please eg "GameDB: X"

**Comment by drachenschanze** [comment]
Hyper Sports 2002 Winter (Japan) 

Capture

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -37266,7 +37266,7 @@ SLPS-25828:
   name: "Pachitte Chonmage Tatsujin 13 - Pachinko Hissatsu Shigotonin III"
   region: "NTSC-J"
 SLPS-25829:
-  name: "A.C.E. Another Century's Episode 2 [Special Vocal Version]"
+  name: "Another Century's Episode 2 [Special Vocal Version]"
```
[the remark this answers is not in the document]
Comment (JordanTheToaster) [reply bin/resources/GameIndex.yaml:37269]: I suggest just not touching it until the others can be confirmed the name is correct as it is in the db so please leave it.

[the remark this answers is not in the document]
Comment (rufotheone-fr) [reply bin/resources/GameIndex.yaml:37269]: @JordanTheToaster : I've forgotten to properly name my commits, sorry... Will do that henceforth.
About Another Century's Episode, every box displays `A.C.E. Another Century's Episode 1/2/3` but every spine only got 'Another Century's Episode` in katakana. When an acronym is used in the japanese title, it should be written on the spine with romajis or katakanas.
But, we have the 2 choices I was talking about (add A.C.E. in every title or remove it).

@drachenschanze Sorry, I don't get the point of your comment?

[the remark this answers is not in the document]
Comment (Tokman5) [reply bin/resources/GameIndex.yaml:37269]: Looking around some Japanese pages, I think A.C.E. should be removed from every title. They don't seem to consider ``A.C.E.`` as a part of full title name.

**Comment by drachenschanze** [comment]
Curious George (Asia)

Capture


http://redump.org/disc/63786/

**Comment by drachenschanze** [comment]
Kan Wo Long Xian Shen Wei (Asia)

image

http://redump.org/disc/26167/

**Comment by F0bes** [comment]
Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)

**Comment by rufotheone-fr** [comment]
> Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)

Mrlinkwii already told me to create a branch then push from there but my master branch was outdated and when I'm trying to sync it with upstream, it wants to create a PR o_O I'm trying to figure out how Github Desktop works but ATM I'm pushing PR, wait for it to be merged then I delete my fork and recreate one u_u How do you keep your master branch synced?

**Comment by rufotheone-fr** [comment]
> Kan Wo Long Xian Shen Wei

I've added this into my PR

**Comment by drachenschanze** [comment]
Keiei Simulation - Jurassic Park (Japan) 

Capture


http://redump.org/disc/40783/

**Comment by JordanTheToaster** [comment]
> Keiei Simulation - Jurassic Park (Japan)
> 
> Capture
> 
> http://redump.org/disc/40783/

Rather than just keep posting here everything you seem to find just make an issue with a list.

**Comment by rufotheone-fr** [comment]
> Keiei Simulation - Jurassic Park (Japan)
> 
> Capture
> 
> http://redump.org/disc/40783/

Added this one too, squashed the commits, properly named it and now?

**Comment by F0bes** [comment]
> > Just an unrelated side note, it's not recommended to push directly to your downstream forks `master` branch. Unless your intent is to provide a different release of PCSX2, it's advised to simply keep your `master` branch in sync with ours, and then branch off of your master when you want to PR a change upstream. :)
> 
> Mrlinkwii already told me to create a branch then push from there but my master branch was outdated and when I'm trying to sync it with upstream, it wants to create a PR o_O I'm trying to figure out how Github Desktop works but ATM I'm pushing PR, wait for it to be merged then I delete my fork and recreate one u_u How do you keep your master branch synced?

Sorry, I don't personally use GitHub desktop, as it's missing crucial features. In this case you probably needed to hard reset your forks master branch. Since GitHub desktop doesn't provide any of that functionality, you would've had to run
`git checkout master`
`git reset --hard origin/master`
This would've cleared any difference between your master and PCSX2's master.

---
Don't do this if you have any PRs **open** pointing to commits on your master branch, as it will remove the commit and might cause issues.

**Comment by rufotheone-fr** [comment]
I'm waiting for this PR to be merged then I'll delete my master branch/recreate one and create a branch for future PR.

So the process is:
- pushing PR from my sub-branch
- wait for it to be merged
(- delete the sub-branch?)
- resync master branch
(- create a new sub-branch and repeat?)

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -12851,7 +12851,7 @@ SLES-51893:
   name: "Naval Ops - Warship Gunner"
   region: "PAL-E"
 SLES-51896:
-  name: "Gallop Racer"
+  name: "Attheraces Presents  Gallop Racer"
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:12854]: Double space after Presents

**Comment by refractionpcsx2** [comment]
Thanks :)

**Comment by rufotheone-fr** [comment]
Thank you too, more coming :)

**Pull request merged**